update_deps/update_deps.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/google/go-containerregistry/pkg/crane"
+)
+
+func main() {
+	grepCmd := exec.Command("git", "grep", "-l", "image: ")
```

Comment:
I'm not a big fan of this kind of logic. What if Buildbarn gains graphics support, thereby adding an `image: ` configuration option to specify a picture? Would we want this code to also process such statements? Clearly not.

Please replace this by a hardcoded list of pathnames where substitutions need to take place.

Reply:
We could also use _git-grep_'s _pathspec_ support doing something like `git grep -l "image: " -- "*.yml" "*.yaml"`, but using a hardcoded list will greatly simplify the code and make it more performant.